### PR TITLE
Render default icon for deleted user avatar

### PR DESCRIPTION
**Pull request by luchaos**

### CODE REVIEW

File: lib/render/user.php [hunk 1]
```diff
@@ -24,14 +24,15 @@ function userAvatar(
...
         $userSanitized = $username;
         sanitize_outputs($userSanitized);
 
-        return '<del>' . $userSanitized . '</del>';
+        $iconLabel = '';
+        if ($icon !== false && ($icon || !$label)) {
+            $iconLabel = "<img src='" . media_asset('/UserPic/_User.png') . "' alt='$userSanitized' width='$iconSize' height='$iconSize'>";
+        }
+
+        return "<span class='inline whitespace-nowrap'>$iconLabel <del>$userSanitized</del></span>";
```
Comment (Jamiras) [inline lib/render/user.php:35]: I don't think you can implicitly put the user name in the text. Suggest putting it in a `title` attribute.
image

Comment (luchaos) [reply lib/render/user.php]: Just noticed 👍

File: lib/render/user.php [hunk 2]
```diff
@@ -24,14 +24,20 @@ function userAvatar(
...
 
-        $userSanitized = $username;
-        sanitize_outputs($userSanitized);
+        $usernameLabel = '';
+        if ($label !== false && ($label || !$icon)) {
+            $usernameLabel = "<del>$userSanitized</del>";
+        }
 
-        return '<del>' . $userSanitized . '</del>';
+        return "<span class='inline whitespace-nowrap'>" . $iconLabel . $usernameLabel . "</span>";
```
Comment (Jamiras) [inline lib/render/user.php:40]: Not sure where the extra padding is coming from, but the images aren't aligned.
image

File: lib/render/user.php [hunk 3]
```diff
@@ -24,14 +24,20 @@ function userAvatar(
     });
 
     if (!$user) {
+        $userSanitized = $username;
+        sanitize_outputs($userSanitized);
+
+        $iconLabel = '';
         if ($icon !== false && ($icon || !$label)) {
-            return '';
+            $iconLabel = "<img src='" . media_asset('/UserPic/_User.png') . "' alt='$userSanitized' width='$iconSize' height='$iconSize'>";
```
Comment (Jamiras) [inline lib/render/user.php:32]: Recommend adding a `title` tag, as the image has no meaning without an associated label or tooltip (code notes).

File: lib/render/user.php (hunk 2 above)
Comment (luchaos) [reply lib/render/user.php]: Thanks, that didn't occur to me.

File: lib/render/user.php (hunk 3 above)
Comment (luchaos) [reply lib/render/user.php]: Good call, added

**Review by Jamiras — APPROVED** [review]

**Pull request merged**